src/metabase/models/interface.clj

```diff
@@ -225,9 +225,18 @@
   "Serialize encrypted json."
   (comp encryption/maybe-encrypt json-in))
 
-(def encrypted-json-out
+(defn encrypted-json-out
   "Deserialize encrypted json."
-  (comp json-out-with-keywordization encryption/maybe-decrypt))
+  [v]
+  (let [decrypted (encryption/maybe-decrypt v)]
+    (try
+      (json/parse-string decrypted true)
+      (catch Throwable e
+        (if (or (encryption/possibly-encrypted-string? decrypted)
+                (encryption/possibly-encrypted-bytes? decrypted))
+          (log/error e "Could not decrypt encrypted field! Have you forgot to set MB_ENCRYPTION_SECRET_KEY?")
```

Comment:
The problem is now that this message will effectively be logged twice when the log level is set to `warn`.

I feel like the correct thing to do here is to skip logging errors in the maybe-decript step, and instead only log errors here. So that means reverting this change you just made (sorry)
https://github.com/metabase/metabase/pull/36701/files#diff-af1ac78232eccb809814746d4ffc7edfc00b17d340caf897b62992384f79acdaR150

Reply:
Do you think it's a problem? :-)

Reply:
I can't think of any downsides other than it's a little strange to have two checks that are almost the same, and logging slightly different messages. I guess we can let this slide for practicality sake. But on the other hand it's easily avoidable

Reply:
Not really easily, except for checking for secret key in `models.interface` which I really don't want to do, or stopping using `maybe-decrypt` in favor of `decrypt` dying. I've tried that, but I feel like it changes behavior too much.